src/Infrastructure.VS.UnitTests/ErrorListHelperTests.cs

```diff
@@ -0,0 +1,194 @@
+﻿/*
```

Comment:
All of this was moved from DisableRuleCommandTests, all I did was adjust them to include csharp and visual basic errorcodes.